admin-manual/installation-setup/installation/install-ubuntu.rst

```diff
@@ -388,9 +388,9 @@ the configuration of your new server.
    ``http://127.0.0.1:8000`` if this is a local development setup). The default
    username and password are ``test``/ ``test``.
 
-   If you are using a public IP address, you will need to configure your
-   firewall rules and allow access only to ports 80 and 8000 for Archivematica
-   usage.
+   If you are using an IP address or fully-qualified domain name instead of
+   localhost, you will need to configure your firewall rules and allow access
```

Comment:
I think it is a typo, it should be: ports 80 and 8000

Reply:
Good catch! Resolved.